jenkins_jobs/trigger_yamllint_from_versions_repo.groovy

```diff
@@ -0,0 +1,43 @@
+job('trigger_yamllint_from_versions_repo') {
+  label('validation_slave_label')
+  concurrentBuild()
+  parameters {
+    stringParam('builds_sha1', 'master', 'SHA-1 of the builds commit.')
+    stringParam('sha1', 'master', 'SHA-1 of the commit to validate.')
```

Comment:
Please be consistent across jobs. If you are allowing manual execution for RPMlint, do ti here too.